src/move_exchange.jl

```diff
@@ -259,3 +180,73 @@ function cost_move_exchange(solution::Solution, car_pos_a::Int, car_pos_b::Int,
 
     return cost_on_objective
 end
+
+"""
+    cost_move_exchange(solution::Solution, car_pos_a::Int, car_pos_b::Int,
+                       instance::Instance, objective::Int)
+
+Returns the cost of the exchange of the car `car_pos_a` with the car `car_pos_b` with
+respect to objectives 1 to `objective`. A negative cost means that the move is
+interesting with respect to objective 1 to `objective`.
+"""
+cost_move_exchange(solution::Solution, car_pos_a::Int, car_pos_b::Int, instance::Instance, objective::Int) =
+    cost_move_exchange(solution, car_pos_a, car_pos_b, instance, [trues(objective) ; falses(3-objective)])
+
+
+            #-------------------------------------------------------#
+            #                                                       #
+            #                   Factorization                       #
+            #                                                       #
+            #-------------------------------------------------------#
+
+"""
+    compute_delta_exchange(solution::Solution, instance::Instance,
+                           car_pos_a::Int, car_pos_b::Int,
+                           first_line::Int, last_line::Int)
+
+For factorization
+"""
+function compute_delta_exchange(solution::Solution, instance::Instance,
+                                car_pos_a::Int, car_pos_b::Int,
+                                first_line::Int, last_line::Int)
```

Comment:
As we discuss this kind of function can be seen as a private function: we should append `_` at the beginning of their name.

This comment is a reminder and the occasion for @Jonathan-Fontaine44 @BenoitLB @BenoitLoger and @melauob to give their opinion if they think otherwise.